core/src/Kokkos_View.hpp

```diff
@@ -670,12 +670,12 @@ class View : public Impl::BasicViewFromTraits<DataType, Properties...>::type {
   // Special function to be preferred over the above for passing in 0, NULL or
   // nullptr when pointer type is char*
   template <class... Args>
-  explicit View(decltype(nullptr), Args... args)
+  KOKKOS_FUNCTION explicit View(std::nullptr_t, Args... args)
       : View(Kokkos::view_wrap(pointer_type(nullptr)), args...) {}
 
   // FIXME: Constructor which allows always 8 sizes should be deprecated
   template <class... P>
-  explicit View(
+  KOKKOS_FUNCTION explicit View(
```

Comment:
This constructor is for a managed View. Why do you want to make it device-callable?
```suggestion
  explicit View(
```

Reply:
I do agree that the use case is quite debatable :wink:

Reply:
Because the View constructor from nullptr above delegate to this constructor.
https://github.com/kokkos/kokkos/blob/bc7d374702a52335b052e68b3b8f78100de19f5e/core/src/View/Kokkos_ViewCtor.hpp#L475-L478

Reply:
No, it delegates to https://github.com/kokkos/kokkos/blob/bc7d374702a52335b052e68b3b8f78100de19f5e/core/src/Kokkos_View.hpp#L717-L749.

Reply:
My bad then.  Fixed.